localtypings/validatorPlan.d.ts

```diff
@@ -1,4 +1,6 @@
-namespace pxt.blocks {
+/// <reference path="./pxtpackage.d.ts" />
+
+declare namespace pxt.blocks {
```

Comment:
Why does the types definition still need a namespace?

Reply:
all of the other typings in localtypings used namespaces, not modules. i just matched the style. might be possible to make it a module, but you'll run into trouble when referencing it in the teachertool since it's outside the src folder